tests/test_invoke.py

```diff
@@ -196,13 +199,18 @@ def test_content_string_check_fragments(reset_results_dictionary):
 # pylint: disable=unused-argument
 # pylint: disable=redefined-outer-name
 def test_content_string_check_fragments_exact(reset_results_dictionary):
-    """Checks that the checking of words works correctly"""
+    """Checks that the checking of exact fragments and regex works correctly"""
```

Comment:
Your tests should ***never*** be added to an unrelated test case. This MUST be a new method!!